fix(version): Update package versions to newer releases.

wayfarer3130 opened the pull request:
Update the versions.  Also want to commit it with a fix message to force  build/release cycle.

igoroctaviano commented:
@wayfarer3130 I tried using this OL version before with SLIM and it has breaking changes. We might be able to merge the other changes but major OL updates would require more effort to test.

igoroctaviano commented on package.json:
```diff
@@ -43,7 +43,8 @@
     "fmt": "standard 'src/**/.js' --fix",
     "lint": "standard 'src/**/*.js'",
-    "clean": "rm -rf ./dist ./node_modules",
+    "clean": "rm -rf ./dist",
```
I would rather stay with a single clean script that does it all. If people are using clean as it is, this could break their expectations.

igoroctaviano reviewed (changes requested):
LGTM, left one comment

wayfarer3130 commented:
@igoroctaviano - I have addressed your comment, as well as created
https://github.com/OHIF/Viewers/pull/4016
that specifically has your changes in it.

igoroctaviano reviewed (approved):
LGTM
